tests/phpunit/tests/meta.php

```diff
@@ -419,6 +419,34 @@ public function test_get_metadata_with_empty_key_nested_array_value() {
 		$this->assertSame( array( $value ), $found['foo'] );
 	}
 
+	/**
+	 * @dataProvider data_get_metadata_non_existent_object_id
+	 */
+	public function test_get_metadata_non_existent_object_id( $expected, $args ) {
+		$this->assertSame( $expected, get_metadata( 'user', ...$args ) );
+	}
+
+	public function data_get_metadata_non_existent_object_id() {
+		return array(
+			'Default values for the $meta_key and $single params' => array(
```

Comment:
Thanks for adding the names @rodrigoprimo. They are acceptable as-is.

I'm just leaving an example below of how to make the data set names could be made more descriptive of _what_ is being tested. Changing the names is not required though for approval of this PR (by me).

```suggestion
			'Return value will be an empty array when only required params are passed' => array(
```

Reply:
Thanks again, @jrfnl. It makes sense to me to make the names more descriptive. I added a new commit.